requirements/edx/base.txt

```diff
@@ -55,7 +55,7 @@ edx-oauth2-provider==1.2.0
 edx-opaque-keys==0.4.0
 edx-organizations==0.4.1
 edx-rest-api-client==1.6.0
-edx-search==0.1.2
+#edx-search==0.1.2
```

Comment:
Why did the requirement is commented but not removed?

Reply:
done